qa/sc_bootstrap.py

```diff
@@ -33,58 +34,37 @@ def setup_nodes(self):
     def sc_setup_chain(self):
         mc_node = self.nodes[0]
         sc_node_1_configuration = SCNodeConfiguration(
-            MCConnectionInfo()
+            MCConnectionInfo(address="ws://{0}:{1}".format(mc_node.hostname, websocket_port_by_mc_node_index(0)))
         )
         sc_node_2_configuration = SCNodeConfiguration(
-            MCConnectionInfo()
+            MCConnectionInfo(address="ws://{0}:{1}".format(mc_node.hostname, websocket_port_by_mc_node_index(0)))
```

Comment:
one sc node is enough for this test, right?

Reply:
Removed sc nodes 2 and 3.